joern-cli/frontends/gosrc2cpg/src/test/scala/io/joern/go2cpg/passes/ast/StructureCallTests.scala

```diff
@@ -0,0 +1,211 @@
+package io.joern.go2cpg.passes.ast
+
+import io.joern.go2cpg.testfixtures.GoCodeToCpgSuite
+import io.shiftleft.semanticcpg.language.*
+
+import scala.collection.immutable.List
+
+class StructureCallTests extends GoCodeToCpgSuite(withOssDataflow = true) {
+
+  "when structure declared with multiple argument" should {
+    val cpg = code(
+      """
+        package main
+        |func main() {
+        |    rect := Rectangle{width: 10, height: "somestr"}
+        |}
+        |""".stripMargin,
+      "test.go"
+    )
+    val List(rectangleCallNode) = cpg.call.nameExact("Rectangle").l
+    "test basic cpg properties for call node of structure declaration" in {
+      rectangleCallNode.name shouldBe "Rectangle"
+      rectangleCallNode.methodFullName shouldBe "main.Rectangle"
```

Comment:
This looks like a constructor to me in some sense, so should we not stick to the convention of calling the constructor method `<init>`?

Reply:
Addressed here: https://github.com/joernio/joern/pull/3596/commits/cb42da630eaa582260c9341c5b345b16895233c7
Added postfix ".<init>" for every structure declaration